la_palma_overview/__init__.py

```diff
@@ -9,53 +9,53 @@
     -v --verbose                tell what is currently done
 
 Notes:
-    - When output is not specified, a time stamp image name is created: 
+    - When output is not specified, a time stamp image name is created:
       'la_palma_yyyymmdd_HHMMSS.jpg'
     - A UTC time stamp and the FACT telescope status are put into the image.
 """
 from __future__ import absolute_import, print_function, division
 
 __all__ = ['save_image']
 
-import matplotlib
-matplotlib.use('Agg') 
-# Must be set to enforce matplotlib runs on machines 
-# where no x server backend is specified
```

Comment:
I was wondering why it was possible to get rid of this. Then I found issue #3 explaining this. 
Would be good to reference PR #4 to issue #3